Ch10 galrep

tscholl2 opened the pull request.

Commit:
fixed bibliography

Commit:
minor changes

changes discussed while reading through

Commit:
E[n] in char p

Added remark on hw E[n] = (Z/nZ)^2 in general, mostly.

williamstein commented on galrep.tex:
```diff
@@ -425,12 +425,22 @@ \subsection{Other Groups}
+	to the case of positive characteristic. Another method is to
+	compute the degree of the multiplication by $n$ map and show
+	it is separable. For more a detailed proof see
+	\cite[Cor.~III.6.4]{silverman:aec}
```
period at end of sentence.

williamstein commented on galrep.tex:
```diff
@@ -425,12 +425,22 @@ \subsection{Other Groups}
 \end{remark}
 
+\begin{remark}
+	The fact that $E[n] \approx (\Z/n\Z)^2$ holds over fields of
```
\approx --> \ncisom    and declare that as a macro.

\isom macro

williamstein replied:
"The fact that $E[n] \approx (\Z/n\Z)^2$ holds over fields of" -->
"In fact, if $p$ is a prime that does not divide $n$, then $E[n] \approx (\Z/n\Z)^2$ over fields of"  (and get rid of "with special exception..."). 

$p>0$ --> $p$.

williamstein commented on galrep.tex:
```diff
@@ -425,12 +425,22 @@ \subsection{Other Groups}
+\begin{remark}
+	The fact that $E[n] \approx (\Z/n\Z)^2$ holds over fields of
+	characteristic $p>0$ as well, with special exceptions
+	when $p|n$. However, the methods we used above will not apply
```
$p|n$ --> $p\mid{}n$.

williamstein replied:
"will not " --> "do not"

williamstein commented on galrep.tex:
```diff
@@ -425,12 +425,22 @@ \subsection{Other Groups}
+	when $p|n$. However, the methods we used above will not apply
+	to the case of positive characteristic. Another method is to
+	compute the degree of the multiplication by $n$ map and show
+	it is separable. For more a detailed proof see
```
"it" could refer to "degree" rather than map.

Commit:
Merge remote-tracking branch 'origin/master' into ch10-galrep

Commit:
ch10: updates

fixed things in last pull request
except for the non-canonical iso macro.
that's open as an issue and will be another PR

Commit:
removed unnecessary comma

I think it was unnecessary?

Commit:
removed include only added by accident

tscholl2 committed:
Update ant.tex
